src/timeline/style.ts

```diff
@@ -34,11 +34,13 @@ export default function styleConstructor(theme: Theme = {}, calendarHeight: numb
       backgroundColor: LINE_COLOR,
       ...appStyle.verticalLine,
       position: 'absolute',
-      height: '105%'
+      height: '105%',
+      ...theme.verticalLine
```

Comment:
Take the style from the 'appStyle' instead of theme, as it includes it and add the default styling